templates/widgets/games.tpl

```diff
@@ -2,5 +2,6 @@
 <span>
   <label>Jocuri</label><br />
   <a href="{$wwwRoot}spanzuratoarea">spânzurătoarea</a><br/>
-  <a href="{$wwwRoot}moara">moara cuvintelor</a>
+  <a href="{$wwwRoot}moara">moara cuvintelor</a><br/>
+  <a href="{$wwwRoot}omleta">omleta cuvintelor</a>
```

Comment:
La mine arata ok in bara de jocuri, ce am observat esta ca bara de jocuri nu apare pe mobil. (sau n-am gasit-o eu).